app/views/games/index.html.erb

```diff
@@ -14,36 +14,13 @@
       <div class="games__text games__text--title">Week <%= @week %></div>
 
       <% @games.each do |game| %>
-        <%= render partial: "partials/game", locals: {game: game, user: current_user } %>
+        <%= render partial: "partials/game", locals: {game: game} %>
```

Comment:
Turns out `current_user` is global so you don't need to pass it into the partial.